src/client/components/VoyageDetails.js

```diff
@@ -16,30 +51,60 @@ function VoyageDetails({
   hsfo,
   ulsfo
 }) {
+  const classes = useStyles();
   return (
-    <div>
+    <Card className={classes.card}>
```

Comment:
Please do not put a card around the entire component. The teaser already has its own card. The result is that we get multiple cards nested inside each other.

Instead wrap the section below it and the selected routes table.

Like this: 

<img width="803" alt="Screen Shot 2019-10-21 at 11 20 52" src="https://user-images.githubusercontent.com/1968912/67193140-19638380-f3f5-11e9-8135-5ec05d55d258.png">

Reply:
Resolved.